ffi/java-idsdk/src/main/java/com/iotics/sdk/identity/jna/JnaSdkApiInitialiser.java

```diff
@@ -5,18 +5,21 @@
 import java.io.File;
 
 public class JnaSdkApiInitialiser {
-    private static String LIB_PATH = new File("./lib/lib-iotics-id-sdk.so").getAbsolutePath();
-    private final String libPath;
+    private static String LIB_NAME = "lib-iotics-id-sdk.so";
```

Comment:
lib name under windows/mac might be different? Does Java not have a some way to imply the ending (like C# does - where you don't provide the suffix)?

Reply:
Ah OK - I get it - you have to otherwise specify your own one via other constructor